python/arcticdb/version_store/_normalization.py

```diff
@@ -600,10 +602,19 @@ def denormalize(self, item, norm_meta):
 
         series = pd.Series() if df.columns.empty else df.iloc[:, 0]
 
-        if norm_meta.common.name:
-            series.name = norm_meta.common.name
+        if hasattr(norm_meta.common, "has_name"):
```

Comment:
Is there a similar problem for dataframe index names, or is that handled correctly?

Reply:
Turns out this already worked, added a test for it though